travispy/entities/_entity.py

```diff
@@ -192,8 +192,7 @@ def _load(cls, infos, session):
                 try:
                     setattr(entity, key, value)
                 except AttributeError:
-                    warn('Unknown {0} attribute {1}'
-                         .format(entity.__class__.__name__, key))
+                    pass
```

Comment:
For now just `import log` then `log.debug('Unknown {0} attribute {1}'.format(entity.__class__.__name__, key))` is enough. Later I do the additional changes.